frontend/app/src/views/settings/DefiSettings.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-card>
+    <v-card-title>Defi Settings</v-card-title>
+    <v-card-text>
+      <h3>Active Defi modules</h3>
+      <p class="text--secondary">
+        Only data for the selected modules will be fetched. If no modules are
+        selected data for all the modules will by fetched.
+      </p>
+      <defi-module-selector />
+      <h3 class="mt-6">Selected accounts</h3>
+      <p class="text--secondary">
+        Only data for the selected accounts will be fetched. If no accounts are
```

Comment:
If no accounts are selected all data will be fetched. -> If no accounts are selected for a protocol, data for all accounts will be fetched for that protocol.

Reply:
Is that how the backend works as well? If we define a module as enabled but do not provide a list of accounts, does the backend enable the module for all accounts? Or would the wizard need to provide all of the accounts "in the background"?

The latter option would cause a lot of confusion because obviously if an account was added then this would need to be re-submitted somehow by the user.

Reply:
>Is that how the backend works as well? If we define a module as enabled but do not provide a list of accounts, does the backend enable the module for all accounts

yes